DAL/pojos/favoriteWorkoutEntity.js

```diff
@@ -0,0 +1,42 @@
+module.exports = class favoriteWorkout {
```

Comment:
Class name must be the first capital letter

- [ ]  Rename the class name to FavoriteWorkout instead.

## Check again the properties for favoriteWorkout table within UML Diagram 
- [ ] favoriteWorkoutId
- [ ] userId
- [ ] tabataId
- [ ] routineId
- [ ] challengeId

the above are the right properties to FavoriteWorkout Class